traefik/tests/traefik-config_test.yaml

```diff
@@ -47,3 +47,32 @@ tests:
       - contains:
           path: spec.template.spec.containers[0].args
           content: "--providers.kubernetesingress.ingressendpoint.publishedservice=foo/bar"
+  - it: should have tls configuration args when specified in values.yaml for port
+    set:
+      ports:
+        websecure:
+          tls:
+            enabled: true
+            options: "foo"
+            certResolver: "certResolverFoo"
+            domains:
+              - main: "sub.domain.tld"
+                sans:
+                  - "alt1.domain.tld"
+                  - "alt2.domain.tld"
+    asserts:
+      - contains:
+          path: spec.template.spec.containers[0].args
+          content: "--entrypoints.websecure.http.tls=true"
+      - contains:
+          path: spec.template.spec.containers[0].args
```

Comment:
Maybe its better to have a "yaml" block in the test?

Reply:
If i am not mistaken, this is to define the name of a tls options configuration, not the options themselves. If that is true this is exactly what it should be